README.md

```diff
@@ -361,6 +362,41 @@ const ColumnAdder = ({ addColumn }) {
 </Board>
 ```
 
+#### `renderCardAdder`
+
+Use this if you want to render your own card adder. You have to pass a function and return your card adder component.
+The function will receive these parameters:
+
+| Arg          | Description                                          |
+| -----------  | ---------------------------------------------------  |
+| `column`     | the column wich the card is being added              |
+
+| function     | Description                                                |
+| ------------ | ---------------------------------------------------------- |
+| `onConfirm`  | Call this function with a new card to add the new card |
```

Comment:
What about to call this `addCard`?

Maybe we should make clear that this will call `onNewCardConfirm` to pass a new `id`? What do you think?

Reply:
Good. I dont think we need to specify about the id, the ```onNewCardConfirm``` already says it